addons/project/views/project_views.xml

```diff
@@ -624,7 +625,6 @@
                                             <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-if="record.rating_last_value.value == 1" title="Latest Rating: Higly Dissatisfied" role="img" aria-label="Sad face"/>
                                         </b>
                                         <!-- formating of the date -->
-                                        <t t-set="date_format" t-value="'MM/DD/YY'" />
```

Comment:
Let's keep it just in case third-party modules use it as well.

Reply:
Done